### PR TITLE
AWS manual deploy

### PR DESCRIPTION
**Pull request by sploiselle**
Deploy CockroachDB on AWS.

@mberhault – Could I get you to review for me?

---
This change is [Reviewable](https://reviewable.io/reviews/cockroachdb/docs/640)

### CODE REVIEW

File: deploy-cockroachdb-on-aws.md [hunk 1]
```diff
@@ -1,12 +1,194 @@
...
+
+	~~~ shell
+	$ ssh <username>@<node1 external IP address>
+	~~~
+
+2.	Install CockroachDB from our latest binary:
+	
+	~~~ shell
+	# Get the latest CockroachDB tarball.
+	$ wget https://binaries.cockroachdb.com/cockroach-{{site.data.strings.version}}.linux-amd64.tgz
```
Comment (mberhault) [inline deploy-cockroachdb-on-aws.md]: should we be using `latest` for the binaries now that we have it?

**Comment by mberhault** [comment]
LGTM

**Comment by sploiselle** [comment]
@jseldess Take a look at this one for me?

---

Review status: 0 of 2 files reviewed at latest revision, 1 unresolved discussion.

---

*[deploy-cockroachdb-on-aws.md, line 86 at r1](https://reviewable.io:443/reviews/cockroachdb/docs/640#-KRFyN8F-_vjD1zitMga-r1-86:-KRcgdn3eqFpeP57DNr8:b-6ww9r7) ([raw file](https://github.com/cockroachdb/docs/blob/6d12a1bb23f8d6299ef2c6e70d1b3a3eb8a17e13/deploy-cockroachdb-on-aws.md#L86)):*
<details><summary><i>Previously, mberhault (marc) wrote…</i></summary>
> should we be using `latest` for the binaries now that we have it?
</details>
Will go through and make a site-wide change after merging this. Thanks for implementing that for us.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/cockroachdb/docs/640#-:-KRcgjinLc0P4o3C_-WC:b-9pawws)*

**Comment by jseldess** [comment]
I'm still not set up on AWS, so I didn't run through the steps myself, but the content :lgtm:.

---

Review status: 0 of 2 files reviewed at latest revision, 1 unresolved discussion.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/cockroachdb/docs/640#-:-KRdCOdB9AA83CVTZRJw:bm986zo)*

**Pull request merged**